test/logger_test.rb

```diff
@@ -455,9 +455,9 @@ class TestLogger < Hanami::Logger; end
           output =
             stub_stdout_constant do
               class TestLogger < Hanami::Logger; end
-              TestLogger.new.info('This is a message with spaces')
+              TestLogger.new.info("This is a message with quotes \" ' and with spaces")
```

Comment:
@jodosha I was wondering about this test, why put an array in the message? I understand a hash, because it could be parsed as json but Array?